### DISCUSSION TITLE
关于跨站点中Waline邮件通知的问题

### DISCUSSION THREAD

**Discussion by jackhanyuan** [walinejs/waline#1629] [Q&A]
有一个问题，我有两个博客网站[sanshui.findn.cn](https://sanshui.findn.cn/)和[findn.cn](https://findn.cn/)。共用了一个waline评论[comment.findn.cn](https://comment.findn.cn/)，现在发现一个问题，由于服务端配置了SITE_NAME 和SITE_URL，结果收到的邮件通知就不对了，另一个网站的评论也会在邮件中提示是在默认网站的评论，通知中的博客链接也不对。有什么方法可以实现吗两个网站共用一个waline评论，邮件通知正常显示吗？另外服务端只存储了url相对链接，共用一个评论的时候也容易链接冲突，是否可以在服务端加一个字段，存储是在哪个site的评论，这样多个网站就可以只配置一个waline评论了。

**Comment by lizheming** [comment]
暂时还不支持多站点，临时的方案是可以部署两个服务，存储服务可以都用一个。
